lab2/index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+    <title>Bitcoin Mining Game</title>
+    <link rel="stylesheet" href="app.css">
+</head>
+<body>
+<!-- bitcoin roughly $8000 02/10/2018 -->
+    <h1>Bitcoin Mining Game</h1>
+    
+    <textarea readonly>
+        Bitcoin? Click on the button to get somes Bitcoin.
+        
+        Your Bitcoin are generating you wealth.
+        
+        B is for Bitcoin. And that is good enough for me.
+    </textarea>
+
+    <div class="clicker">
+      <h2>Amount of Bitcoin (BTC):</h2>
+      <button onclick="buttonClick();">Generate 0.01BTC</button>
+	  <input readonly ddtype="text" id="inc" value="0"></input>
```

Comment:
I see you are using readonly field here for preventing user to update the input content. Good use 💯